src/Compilers/Core/Portable/WellKnownMembers.cs

```diff
@@ -3273,6 +3273,7 @@ internal static bool IsSynthesizedAttributeOptional(WellKnownMember attributeMem
                 case WellKnownMember.System_STAThreadAttribute__ctor:
                 case WellKnownMember.System_Runtime_CompilerServices_AsyncStateMachineAttribute__ctor:
                 case WellKnownMember.System_Runtime_CompilerServices_IteratorStateMachineAttribute__ctor:
+                case WellKnownMember.System_Runtime_CompilerServices_TupleElementNamesAttribute__ctorTransformNames:
                     return true;
```

Comment:
Will remove when dealing with [https://github.com/dotnet/roslyn/issues/13940](https://github.com/dotnet/roslyn/issues/13940)

---
In reply to: [79718272](https://github.com/dotnet/roslyn/pull/13923#discussion_r79718272) [](ancestors = 79718272,79713713,79673650)